<p>
  This rule verifies if some given Java packages or classes are used in a JSP file.
  The following code illustrates this rule, with <code>java.sql,ArrayList</code> as libraries:</p>

<h2>Noncompliant Code Example</h2>

<pre>
&lt;%@ page import="java.sql.*" %&gt;     &lt;!-- Non-Compliant --&gt;
&lt;% java.util.ArrayList clients; %&gt;  &lt;!-- Non-Compliant --&gt;
&lt;% java.lang.String name; %&gt;        &lt;!-- Compliant --&gt;
</pre>
